.teamcity/common/BuildCache.kt

```diff
@@ -20,18 +20,19 @@ interface BuildCache {
     fun gradleParameters(os: Os): List<String>
 }
 
-data class RemoteBuildCache(val url: String, val username: String = "%gradle.cache.remote.username%", val password: String = "%gradle.cache.remote.password%") : BuildCache {
+data class RemoteBuildCache(val url: String,
+                            val username: String = "%gradle.cache.remote.public.username%",
```

Comment:
Does that mean that after this PR, all developers need to switch to use the public build cache instance?

Reply:
No, this is just for transition from `e.grdev.net` to `ge.gradle.org`. After this is merged, we'll keep this config for some time until all developer branches catch up. Then we switch to old config name.

Reply:
I don't understand. After this is merged, CI will push to the new cache location. So if developers want to get some cache hits, they will need to use the new cache URL, right?

Reply:
I see your point. I already changed global cache configuration to `ge.gradle.org` and sent an email ton engineering maillist.